src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -2423,26 +2269,23 @@ protected virtual bool ShouldParseMethodBody (MethodDefinition method)
 			}
 		}
 
-		internal protected void MarkProperty (PropertyDefinition prop, in DependencyInfo reason)
+		internal protected void MarkProperty (PropertyDefinition prop)
```

Comment:
Similar comment as above - won't this introduce a "hole" in the graph? That is we will mark the custom attribute on the property as "reason - property", but there won't be any record of why the property itself was marked...